src/Bridges/Constraint/ltgt_to_interval.jl

```diff
@@ -0,0 +1,87 @@
+# The code here is mostly copied from the flip_sign.jl code for FlipSignBridge and GreaterToLessBridge
+
+"""
+    AbstractToIntervalBridge{T, S1, F, G}
+
+Bridge a `G`-in-`S1` constraint into an `F`-in-`Interval` constraint by multiplying
+the function by `+1` and taking the point reflection of the set across the
+origin. The flipped `F`-in-`S` constraint is stored in the `constraint`
+field by convention.
+"""
+abstract type AbstractToIntervalBridge{
+    T, S1<:MOI.AbstractSet, S2<:MOI.AbstractSet,
+    F<:MOI.AbstractFunction, G<:MOI.AbstractFunction} <: SetMapBridge{T, S2, S1, F, G} end
+
+# The function map is the identity
+map_function(::Type{<:AbstractToIntervalBridge{T}}, func) where {T} = MOIU.operate(+, T, func)
```

Comment:
You can pass the `func` as it is. No need for any operation

Reply:
done